backend/app/notify/notify.go

```diff
@@ -94,25 +100,32 @@ func (s *Service) Submit(req Request) {
 	}
 }
 
-// getNotificationEmails returns list of emails for notifications for provided comment.
-// Emails is not added to the returned list in case original message is from the same user as the notification receiver.
-func (s *Service) getNotificationEmails(req Request, notifyComment store.Comment) (result []string) {
+// getNotificationTargets returns list of notification targets (like email or telegram username) for users
```

Comment:
Updated the comment to reflect the new name, made results deduplicated to prevent the caller from doing it.